src/Microsoft.Identity.Web/TokenCacheProviders/CacheSerializerHints.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+
+namespace Microsoft.Identity.Web.TokenCacheProviders
+{
+    /// <summary>
+    /// todo.
+    /// </summary>
+    public class CacheSerializerHints
```

Comment:
+1 to exposing an object that makes sense for M.I.W., not the lower level object that MSAL exposes (CacheNotificationArgs)